packages/react-sdk/src/components/link.ws.tsx

```diff
@@ -26,6 +26,7 @@ export const meta: WsComponentMeta = {
   label: "Link Text",
   Icon: LinkIcon,
   presetStyle,
+  states: ["[aria-current=page]"],
```

Comment:
I think we should be able to add a label there that would be visible in UI

Reply:
Default humanizer is not good enough?